.circleci/config.yml

```diff
@@ -170,6 +176,7 @@ workflows:
           requires:
             - cross-compile
             - correctness
```

Comment:
Should this be renamed to `correctness_trace`?

Reply:
Yeah, I think so, but was thinking in a separate PR.

Reply:
Ok